Installation fails with Cordova 7: package.json missing

I cannot install the plugin on my setup.

After some digging, I think it is due to the lack of a `package.json` manifest file. Indeed, [Cordova 7's changelog ](https://cordova.apache.org/news/2017/05/04/cordova-7.html)explicitly state:
> Platforms and plugins are now required to have a package.json file.

Here what I get when I try to install it with http:
```
> cordova plugin add https://github.com/fabiorogeriosj/cordova-plugin-sensors.git
Error: Failed to fetch plugin https://github.com/fabiorogeriosj/cordova-plugin-sensors.git via registry.
Probably this is either a connection problem, or plugin spec is incorrect.
Check your connection and plugin name/version/URL.
Error: npm: Command failed with exit code 1 Error output:
npm ERR! cb() never called!
```
Do not say much (and my connection is functional, I am able to install other plugins like `cordova-plugin-multidex` or `cordova-plugin-whitelist`).

When I try with ssh I get:
```
> cordova plugin add git@github.com:fabiorogeriosj/cordova-plugin-sensors.git
Error: Failed to fetch plugin git@github.com:fabiorogeriosj/cordova-plugin-sensors.git via registry.
Probably this is either a connection problem, or plugin spec is incorrect.
Check your connection and plugin name/version/URL.
Error: npm: Command failed with exit code 1 Error output:
npm ERR! code ENOLOCAL
npm ERR! Could not install from "github.com:fabiorogeriosj/cordova-plugin-sensors.git" as it does not contain a package.json file.
```
which does seem to imply that the lack of `package.json` is the reason the install is failing.

--- a/package.json
+++ b/package.json
@@ -1,0 +1,23 @@
+{
+  "name": "cordova-plugin-sensors",
+  "version": "0.7.0",
+  "description": "This plugin gives Cordova's applications access to Android's sensors",
+  "cordova": {
+    "id": "com.fabiorogeriosj.sensors",
+    "platforms": ["android"]
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/QuentinRoy/cordova-plugin-sensors.git"
+  },
+  "keywords": ["ecosystem:cordova", "cordova-android", "sensors"],
+  "engines": {
+    "cordova": ">=3.4.0"
+  },
+  "author": "Fábio Rogério SJ",
+  "license": "",
+  "bugs": {
+    "url": "https://github.com/fabiorogeriosj/cordova-plugin-sensors/issues"
+  },
+  "homepage": "https://github.com/fabiorogeriosj/cordova-plugin-sensors#readme"
+}

